apps/settings/index.html

```diff
@@ -31,6 +31,7 @@
     <link rel="stylesheet" type="text/css" href="shared/style/action_menu.css"/>
     <link rel="stylesheet" type="text/css" href="shared/style/confirm.css"/>
     <link rel="stylesheet" type="text/css" href="shared/style/progress_activity.css"/>
+    <link rel="stylesheet" type="text/css" href="/shared/elements/gaia-icons/bidi-helper.css">
```

Comment:
Related to my previous comment, but will it lazy loaded if I put it outside the comments?